directory/src/main/java/org/ecclesia/directory/domain/Organization.java

```diff
@@ -3,21 +3,25 @@
 import javax.persistence.*;
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck
**Severity**: WARN
**File**: /var/jenkins_home/workspace/ecclesia_PR-6/directory/src/main/java/org/ecclesia/directory/domain/Organization.java L3

Using the &#39;.*&#39; form of import should be avoided - javax.persistence.*.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a616772467>*